README.md

```diff
@@ -1,100 +1,137 @@
-# OSTIS-web-platform
-
+# OSTIS Web Platform
 <img src="https://github.com/ostis-ai/ostis-web-platform/actions/workflows/main.yml/badge.svg?branch=develop">
 
-## Documentation:
+This repository is a web-oriented software platform of [OSTIS Project](https://github.com/ostis-ai/ostis-project), and it is intended to be a solid framework to help you deploy existing and create new OSTIS systems.
 
-This repository represents ***the OSTIS Technology web-oriented platform*** implementation. You can use it to store, represent 
-and process your information in formal language of universal representation of knowledge. The OSTIS Technology platform 
-is associated with the IMS system. 
+OSTIS Web platform contains:
 
-The OSTIS Technology platform contains:
-1) [Knowledge base with top-level ontologies](https://github.com/ostis-ai/ims.ostis.kb) to specify any information models 
-in scope of any subject domains;
-2) [Knowledge processing machine](https://github.com/ostis-ai/sc-machine) with semantic network storage and their 
-processing with tools to interact with it;
-3) [Web-oriented semantic interface](https://github.com/ostis-ai/sc-web) to users interact with the IMS system.
+1) [Knowledge base with top-level ontologies](https://github.com/ostis-ai/ims.ostis.kb) to help you develop a broad variety of information models
+2) [Knowledge processing machine](https://github.com/ostis-ai/sc-machine) with semantic network storage and agent-based knowledge processing
+3) [Web-oriented semantic interface](https://github.com/ostis-ai/sc-web) for users to interact with the intelligent system.
 
-# Installation
-## Using Docker
-You will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+To learn more about the platform, check out our [documentation](https://raw.githubusercontent.com/ostis-ai/ostis-web-platform/develop/docs/main.pdf).
```

Comment:
this link push me to download pdf file. I think, better to use https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf